experimental/orchestrator/README.md

```diff
@@ -1,37 +1,55 @@
-2. **Recognizer**: You can use Orchestrator as a recognizer with [Adaptive dialogs][6]. 
+## Scenarios
+**Dispatch**: Orchestrator is a successor to [dispatch][5]. You can use Orchestrator instead of the current [dispatch][5] solution to arbitrate across your [LUIS][1] and [QnA Maker][2] applications. With Orchestrator, you are likely to see:
 
-## Authoring experience
+- Improved classification accuracy
+- Higher resilience to data imbalance across your LUIS and QnA Maker authoring data.
+- Ability to correctly dispatch from relatively little authoring data.
 
-Here's the end to end authoring experience.
+**Intent Recognizer**: You can use Orchestrator as an intent recognizer with [Adaptive dialogs][6]. Using the same approach as in the dispatch scenario above in order to route to responses within your bot logic.
+
+**Entity Extraction** is not supported yet.  It is on the planning roadmap to add entity extraction in the future.
+
+## Authoring Experience
+
+Orchestrator can be used in different development environments:
+
+* **Code First**: Orchestrator can be integrated into your code project by replacing luis and/or qnamaker for language understanding.  See [Runtime Integration][# Runtime Integration] section for more.
+* [Bot Framework Composer][19]: Orchestrator can be selected as a recognizer within Bot Framework Composer. At this point there are limitations to using Orchstrator in Composer primarily around importing of existing models and tuning up recognition performance (* currently available only if the feature flag is enabled with Composer).
+
+Thus, use of [BF command line tool][7] to prepare and optimize the model for your domain is required in most, if note all, use cases.   To illustrate the workflow, here is a sample the end to end authoring experience:
 
 <p align="center">
   <img width="350" src="./docs/media/authoring.png" />
 </p>
 
-### Using [BF CLI][7]
+### Prepare
+
+* Pre-requisite: Install [BF CLI Orchestrator plugin][11] first.
 
-* Pre-requisite: Install [BF CLI Orchestrator plugin][11] first. 
+1. Author Intent-utterances example based .lu definition referred to as a *label file* using the Language Understanding practices as described in [Language Understanding][2] for dispatch (e.g. author .lu file or within the [Composer][3] GUI experience).
+   * Alternatively, [export][8] your LUIS application and [convert][9] to .lu format or [export][10] your QnA Maker KB to .qna format 
+2. Download Natural Language Representation ([NLR][20]) base Model (will be referred to as the *basemodel*) using the `bf orchestrator:basemodel:get` command. 
+   * See `bf orchestrator:basemodel:list` for alternate models. You may need to experiment with the different models to find which performs best for your language domain.
+3. Combine the label file .lu from (1) with the base model from (2) to create a *snapshot* file with a .blu extension.
+   * Use [`bf orchestrator:create`][16] to create just the .blu snapshot file.
```

Comment:
This is dispatch scenario mostly, one snapshot file for all Lu/json/qna tsv files.